test/jdk/java/security/cert/CertStore/NoLDAP.java

```diff
@@ -25,20 +25,23 @@
  * @bug 8004502
  * @summary Sanity check that NoSuchAlgorithmException is thrown when requesting
  *   a CertStore of type "LDAP" and LDAP is not available.
+ * @library /test/lib
  */
 
 import java.security.NoSuchAlgorithmException;
 import java.security.cert.CertStore;
 import java.security.cert.LDAPCertStoreParameters;
+import jtreg.SkippedException;
 
 
 public class NoLDAP {
     public static void main(String[] args) throws Exception {
         try {
             Class.forName("javax.naming.ldap.LdapName");
-            System.out.println("LDAP is present, test skipped");
```

Comment:
Maybe we can add a new line to throw a Exception when the expected ClassNotFoundException do not seen.

```java
throw new RuntimeException("The expexted ClassNotFoundException was not observed.");
```